webapp/components/notifications/Notification/Notification.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <ds-space :class="[{ read: notification.read }, notification]" margin-bottom="x-small">
+    <no-ssr>
+      <ds-space margin-bottom="x-small">
+        <hc-user
+          v-if="resourceType == 'Post'"
+          :user="post.author"
+          :date-time="post.createdAt"
+          :trunc="35"
+        />
+        <hc-user v-else :user="comment.author" :date-time="comment.createdAt" :trunc="35" />
+      </ds-space>
+      <ds-text color="soft">
+        {{ $t('notifications.menu.mentioned', { resource: resourceType }) }}
+      </ds-text>
+    </no-ssr>
+    <ds-space margin-bottom="x-small" />
+    <nuxt-link
+      class="notification-mention-post"
+      :to="{ name: 'post-id-slug', params, ...hashParam }"
+      @click.native="$emit('read')"
+    >
+      <ds-space margin-bottom="x-small">
+        <ds-card
+          :header="post.title || comment.post.title"
+          hover
+          space="x-small"
+          class="notifications-card"
+        >
+          <ds-space margin-bottom="x-small" />
+          <div v-if="resourceType == 'Post'">{{ post.contentExcerpt | removeHtml }}</div>
+          <div v-else>
+            <span class="comment-notification-header">Comment:</span>
+            {{ comment.contentExcerpt | removeHtml }}
+          </div>
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-08-19T14:51:22Z" title="Monday, August 19th 2019, 4:51:22 pm +02:00">Aug 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,89 @@
+<template>
+  <ds-space :class="{ notification: true, read: notification.read }" margin-bottom="x-small">
+    <no-ssr>
+      <ds-space margin-bottom="x-small">
+        <hc-user
+          v-if="resourceType == 'Post'"
+          :user="post.author"
+          :date-time="post.createdAt"
+          :trunc="35"
+        />
+        <hc-user v-else :user="comment.author" :date-time="comment.createdAt" :trunc="35" />
+      </ds-space>
+      <ds-text color="soft">
+        {{ $t('notifications.menu.mentioned', { resource: resourceType }) }}
+      </ds-text>
+    </no-ssr>
+    <ds-space margin-bottom="x-small" />
+    <nuxt-link
+      class="notification-mention-post"
+      :to="{ name: 'post-id-slug', params, ...hashParam }"
+      @click.native="$emit('read')"
+    >
+      <ds-space margin-bottom="x-small">
+        <ds-card
+          :header="post.title || comment.post.title"
+          hover
+          space="x-small"
+          class="notifications-card"
+        >
+          <ds-space margin-bottom="x-small" />
+          <div v-if="resourceType == 'Post'">{{ post.contentExcerpt | removeHtml }}</div>
+          <div v-else>
+            <b>
+              Comment: &nbsp;
+            </b>
```


You could make this a `<span>` and give it a class (with `font-weight` and `margin-right`) to get rid of the `&nbsp;` 😉

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-20T05:36:57Z" title="Tuesday, August 20th 2019, 7:36:57 am +02:00">Aug 20, 2019</time>_

---


That’s okay with me, in case @mattwr18 likes to do it, because he will be responsible for today … @alina-beck